app/src/components/TextInput.tsx

```diff
@@ -1,19 +1,20 @@
+import { Trans } from 'next-i18next'
 import { ChangeEvent, ReactElement } from 'react'
 
 type Props = {
   handleChange: (e: ChangeEvent<HTMLInputElement>) => void
   id: string
-  label: string
+  labelKey: string
   required?: boolean
   value: string
 }
 
 const TextInput = (props: Props): ReactElement => {
-  const { handleChange, id, label, required, value } = props
+  const { handleChange, id, labelKey, required, value } = props
   return (
     <>
       <label className="usa-label" htmlFor={id}>
-        {label}
+        <Trans i18nKey={labelKey} />
         {required && <abbr className="usa-hint usa-hint--required"> *</abbr>}
```

Comment:
@rocketnova would it be worth the effort to create a small reusable component with: 

`<abbr className="usa-hint usa-hint--required"> *</abbr>`

That TextArea and RequiredQuestionStatement would also use?

Reply:
Yes! Great call. I've split that out into a separate ticket: https://wicmtdp.atlassian.net/browse/WMDP-250